generator/generator-opc-generator/generators/app/index.js

```diff
@@ -0,0 +1,100 @@
+// @ts-nocheck
+"use strict";
+const Generator = require("yeoman-generator");
+const chalk = require("chalk");
+const yosay = require("yosay");
+const _ = require("lodash");
+
+module.exports = class extends Generator {
+  prompting() {
+    // Have Yeoman greet the user.
+    this.log(
+      yosay(`Welcome to the super ${chalk.red("opc-generator")} generator!`)
+    );
+
+    const prompts = [
+      {
+        type: "input",
+        name: "componentName",
+        message: "Component Name? ",
+        default: "opc-my-component"
+      },
+      {
+        type: "input",
+        name: "componentDescription",
+        message: "Briefly Describe your component: ",
+        default: "Web component based on lit-html"
+      },
+      {
+        type: "input",
+        name: "authorName",
+        message: "Author Name: ",
+        default: this.user.git.name()
+      },
+      {
+        type: "input",
+        name: "authorEmail",
+        message: "Author Email: ",
+        default: this.user.git.email()
+      }
+    ];
+
+    return this.prompt(prompts).then(props => {
+      // To access props later use this.props.someAnswer;
```

Comment:
@hybridx Either remove or update this comment to `// To access props later use this.props;`

Reply:
Changed both